indico/modules/events/timetable/templates/timetable_pdf_export.html

```diff
@@ -17,6 +17,21 @@
     {% endcall %}
     {% call form_footer(form) %}
         <input class="i-button big highlight" type="submit" value="{% trans %}Generate{% endtrans %}">
-        <button type="button" class="i-button big" data-button-back>{% trans %}Back{% endtrans %}</button>
+        <a href="{{ back_url }}" class="i-button big">{% trans %}Back{% endtrans %}</a>
     {% endcall %}
+    <script>
+        $('[name=document_settings][value=showCoverPage], [name=document_settings][value=showTableContents], ' +
```

Comment:
at least specify some parent selector (e.g. the form).. querying fields globally by nothing but their name like this is pretty ugly